queries/boundaries.jinja2

```diff
@@ -84,6 +85,7 @@ SELECT
   'yes' AS maritime_boundary,
   NULL AS admin_level,
   NULL AS boundary_type,
+  NULL AS kind,
```

Comment:
We only want to fold in a [specific value](https://github.com/mapzen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/transform.py#L378) from `boundary:type` into `kind`. We could put it in a `CASE WHEN ... END AS kind` statement, but since it's already implemented in Python I figured it eas just easier to keep the queries the way they are for the moment. What do you think?